app.rb

```diff
@@ -0,0 +1,5 @@
+require 'sinatra'
+
+get '/' do
+  send_file 'public/index.html'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.